gcam_metadata.xml

```diff
@@ -0,0 +1,72 @@
+  <identifier identifierType="DOI">10.5072/example-full</identifier>
+  <creators>
+    <creator>
+      <creatorName nameType="Personal">Person, I.M.</creatorName>
+      <givenName>I.M.</givenName>
+      <familyName>Person</familyName>
+      <nameIdentifier schemeURI="http://orcid.org/" nameIdentifierScheme="ORCID">0000-0001-5000-0007</nameIdentifier>
+      <affiliation affiliationIdentifier="https://ror.org/04wxnsj81" affiliationIdentifierScheme="ROR">DataCite</affiliation>
+    </creator>
+    <creator>
+      <creatorName nameType="Organizational" xml:lang="en">Pacific Northwest National Laboratory</creatorName>
+    </creator>
+  </creators>
+  <titles>
+    <title xml:lang="en-US">Full DataCite XML Example</title>
+    <title xml:lang="en-US" titleType="Subtitle">Demonstration of DataCite Properties.</title>
+  </titles>
+  <publisher xml:lang="en">Pacific Northwest National Laboratory</publisher>
+  <publicationYear>2020</publicationYear>
+  <subjects>
+    <subject xml:lang="en-US" schemeURI="http://dewey.info/" subjectScheme="dewey">000 computer science</subject>
+  </subjects>
+  <contributors>
+    <contributor contributorType="ProjectLeader">
+      <contributorName>Human, I.M.</contributorName>
+      <givenName>I.M.</givenName>
+      <familyName>Human</familyName>
+      <nameIdentifier schemeURI="http://orcid.org/" nameIdentifierScheme="ORCID">0000-0002-7285-027X</nameIdentifier>
+      <affiliation affiliationIdentifier="https://ror.org/03yrm5c26" affiliationIdentifierScheme="ROR">University of Maryland</affiliation>
+    </contributor>
+    <contributor contributorType="Sponsor">
+      <contributorName xml:lang="en">University of Maryland</contributorName>
+    </contributor>
+    <contributor contributorType="Producer">
+      <contributorName xml:lang="en">Pacific Northwest National Laboratory</contributorName>
+    </contributor>
+  </contributors>
+  <dates>
+    <date dateType="Updated" dateInformation="Updated with 4.3 properties">2020-06-02</date>
+  </dates>
+  <language>en-US</language>
+  <resourceType resourceTypeGeneral="Software">XML</resourceType>
+  <alternateIdentifiers>
+    <alternateIdentifier alternateIdentifierType="URL">https://schema.datacite.org/meta/kernel-4.3/example/datacite-example-full-v4.3.xml</alternateIdentifier>
+  </alternateIdentifiers>
+  <relatedIdentifiers>
+    <relatedIdentifier relatedIdentifierType="URL" relationType="HasMetadata" relatedMetadataScheme="citeproc+json" schemeURI="https://github.com/citation-style-language/schema/raw/master/csl-data.json">https://data.datacite.org/application/citeproc+json/10.5072/example-full</relatedIdentifier>
+  </relatedIdentifiers>
+  <sizes>
+    <size>4 kB</size>
+  </sizes>
+  <formats>
+    <format>application/xml</format>
+  </formats>
+  <version>4.3</version>

```

Comment:
I think some people will use non-release versions of the model. Maybe we can add a tag with `modifications-from-release` and possibly a git has. Also, it might be worth having a tag on `scenario-features` where the user can specify which optional features were enabled in the scenario data (e.g., carbon price, portfolio-standard, etc.)

Reply:
@kvcalvin this is a fantastic idea!  Thank you!